eng/Publishing.props

```
@@ -1,11 +1,15 @@
-<Project InitialTargets="ValidateDownloadedAssets">
+<Project>
+  <Import Project="$(MSBuildThisFileDirectory)OSArch.props" />
+  <Import Project="$(MSBuildThisFileDirectory)RuntimeIdentifier.props" />
 
   <PropertyGroup>
     <ProducesDotNetReleaseShippingAssets>true</ProducesDotNetReleaseShippingAssets>
     <!-- This avoids creating VS.*.symbols.nupkg packages that are identical to the original package. -->
     <AutoGenerateSymbolPackages>false</AutoGenerateSymbolPackages>
-    <!-- Set PlatformName to TargetArchitecture to create unique build manifest files. -->
-    <PlatformName Condition="'$(TargetArchitecture)' != ''">$(TargetArchitecture)</PlatformName>
+    <!-- Except for jobs that set a specific asset manifest file name as a global property, set the manifest name based on the package RID. -->
+    <AssetManifestFileName>$(PackageRID).xml</AssetManifestFileName>
```

As mentioned in another comment, consider using the AGENT_JOBNAME env var here. This should into Arcade ideally:

```suggestion
    <AssetManifestFileName Condition="'$(AGENT_JOBNAME)' != ''">$(AGENT_JOBNAME)</AssetManifestFileName>
```